docs/en/plugins/cherrypicker.md

```diff
@@ -42,6 +42,7 @@ In addition to implementing the core functionality of cherry-pick, it also suppo
 | label_prefix             | string   | The prefix of the label that triggers cherry-pick, default is `cherrypick/`                                                                      |
 | picked_label_prefix      | string   | The label prefix of the PR created by cherry-pick (e.g. `type/cherry-pick-for-release-5.0`)                                                      |
 | exclude_labels           | []string | Some labels that you don't want to be automatically copied by the plugin (e.g. some labels that control code merging)                            |
+| auto_request_review      | bool     | Whether to request reviewers of the original PR on the PR created by cherry-picker for review, not automatically requested by default            |
```

Comment:
Do we really need this cfg? Because blunderbuss is still requesting reviewers for the PR of the release branch.

So I think the right thing to do might be to just remove the feature and make blunderbuss support branch configuration?

Reply:
But, these two automatic request review modes have different ways. CherryPicker requests the reviewers of the original PR. Blunderbuss is picked up the reviewers from a list and requested them.

If we want to remove a feature, I think we should ask the person who initially wanted the feature.

Reply:
But our core purpose is to reduce distribution? If the distribution still continues, then the change actually has little effect?

Reply:
OK, I'm worried about there are some repositories that need this feature, so keep it as an option. Maybe we should ask the maintainers?

Reply:
Maybe we can use this option as a transition. If no one needs this feature for a long time, we will remove it completely.

Reply:
> Maybe we should ask the maintainers?

I think we need a discussion.

Reply:
> I think we need a discussion.

@zhouqiang-cl 's suggestion is to remove this feature directly.